globals.go

```diff
@@ -415,6 +415,29 @@ func PatchStatusToVersionStatus(patchStatus string) (string, error) {
 	}
 }
 
+// Constants for pull request comments.
+const (
+	RetryComment   = "evergreen retry"
+	PatchComment   = "evergreen patch"
+	TriggerComment = "evergreen merge"
+)
+
+func trimComment(comment string) string {
+	return strings.Join(strings.Fields(strings.ToLower(comment)), " ")
+}
+
+func IsRetryComment(comment string) bool {
```

Comment:
One is used in model/commitqueue/commit_queue.go and one is used in rest/route/github.go. globals felt like the best neutral place to put it especially considering that we plan to add to it and it would be good to have them all in one place.

Reply:
TriggersCommitQueue is in model but really shouldn't be since it's used by rest/route/github only; I think even going forward any comment parsing should be limited to rest/route/github since that's where we should decide whether to move forward or not.

Reply:
very true. updated.